### PR TITLE
Create stack.cpp

### PR DESCRIPTION
**Pull request by zeolee01**

### CODE REVIEW

File: Algorithms/insertion sort [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include<iostream>
```
Comment (harshit-sharma-gits) [inline Algorithms/insertion sort:1]: Add you name, email in the format mentioned in #12 

Also you haven't added the file extension for this file. rename it to `insertionsort.cpp`

File: Data Structures/Linked List/stack.cpp [hunk 2]
```diff
@@ -0,0 +1,119 @@
+/*
```
Comment (harshit-sharma-gits) [inline Data Structures/Linked List/stack.cpp:1]: Again, add you name, email here in the first line. As mentioned in #12

**Review by harshit-sharma-gits — CHANGES_REQUESTED** [review]
I've added some review, please check

File: Data Structures/Linked List/stack.cpp (hunk 2 above)
Comment (zeolee01) [reply Data Structures/Linked List/stack.cpp:1]: Contributed by - HARSH AGARWAL <harsh5nov11@gmail.com>

**Review by zeolee01 — COMMENTED** [review]
Contributed by - HARSH AGARWAL <harsh5nov11@gmail.com>

**Review by harshit-sharma-gits — APPROVED** [review]
Now your changes are looking good to me @zeolee01 
Approving!

**Pull request merged**